tests/test_main.py

```diff
@@ -55,18 +88,72 @@ def test_create_job_description_with_team(basic_config):
     assert description == "-test-dir #test-team"
 
 
-def test_expand_vars_user(basic_config):
-    from os import environ
+def test_expand_vars_and_user(expandable_struct, caplog):
```

Comment:
Thanks for adding this test!

For future reference, it’s usually better to split large test cases into smaller, atomic tests. Smaller tests are easier to maintain and debug.